### PR TITLE
[13.0][ADD] stock_picking_product_availability_inline: new module.

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT37895

This module extends the functionality of 'Warehouse Management to show you the stock availability (Free To Use Quantity) of each product as part of its name in the product drop-down widget on stock picking line (Operations).

### CODE REVIEW

**Comment by ernestotejeda** [comment]
ping @sergio-teruel @pedrobaeza

File: stock_picking_product_availability_inline/models/stock_picking.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    _inherit = "stock.picking"
+
+    picking_type_warehouse_id = fields.Many2one(
+        comodel_name="stock.warehouse",
+        string="Stock picking type warehouse",
+        related="picking_type_id.warehouse_id",
+        store=True,
+        readonly=False,
+    )
+
```
Comment (sergio-teruel) [inline stock_picking_product_availability_inline/models/stock_picking.py:17]: Why do you not use this field https://github.com/odoo/odoo/blob/b6833702044b0d3057d543dae71fbea7ef091d98/addons/stock/models/stock_picking.py#L50?

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by sergio-teruel** [comment]
Test are failing due to freezegun dependency...
Already in requirements.txt file

**Review by legalsylvain — APPROVED** [review]
Thanks. Great feature. 

Lgtm, code review / no test.

Nice to have / not blocking could you add a screenshot and a référence to the screenshot in the readme ?

File: stock_picking_product_availability_inline/models/stock_picking.py (hunk 1 above)
Comment (ernestotejeda) [reply stock_picking_product_availability_inline/models/stock_picking.py:17]: This field doesn't store the picking type warehouse. Isn't the idea to work with the 'picking type' warehouse?

**Comment by ernestotejeda** [comment]
> Test are failing due to freezegun dependency... Already in requirements.txt file

The failing module is stock_picking_group_by_partner_by_carrier_by_date. Shouldn't that module also have the freezegun dependency in the readme? I don't remember if that is nessesary

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**